app/models/manageiq/providers/base_manager/refresher.rb

```diff
@@ -23,6 +23,11 @@ def refresher_options
         options[self.class.ems_type]
       end
 
+      # Rhev specific method
+      def self.get_target(event_hash)
+        self.class.try(:get_target, event_hash)
```

Comment:
Yes that's perfect as long as the body of the method is implemented per provider.